hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -179,19 +194,58 @@ public static void main(final String... args) throws Exception {
         final SoftwareVersion version = hedera.getSoftwareVersion();
         logger.info("Starting node {} with version {}", selfId, version);
 
-        final PlatformBuilder platformBuilder = PlatformBuilder.create(
-                Hedera.APP_NAME,
-                Hedera.SWIRLD_NAME,
+        final var configuration = buildConfiguration();
+        final var keysAndCerts =
+                initNodeSecurity(bootstrapAddressBook, configuration).get(selfId);
+
+        setupGlobalMetrics(configuration);
+        final var metrics = getMetricsProvider().createPlatformMetrics(selfId);
+        final var time = Time.getCurrent();
+        final var fileSystemManager = FileSystemManager.create(configuration);
+        final var recycleBin =
+                RecycleBin.create(metrics, configuration, getStaticThreadManager(), time, fileSystemManager, selfId);
+
+        final var cryptography = CryptographyFactory.create();
+        CryptographyHolder.set(cryptography);
+        // the AddressBook is not changed after this point, so we calculate the hash now
+        cryptography.digestSync(bootstrapAddressBook);
```

Comment:
`digestSync` will return the calculated hash, but we ignore it, why is that?
Do we call the method for some side effect from it?

Reply:
@MiroslavGatsanoga , I think it sets hash on the addressBook MerkleNode, since it is part of state.